### DISCUSSION TITLE
Comment faire un Pull request ?

### DISCUSSION THREAD

**Discussion by Ouragann** [SOZ-Faut-etre-Sub/SOZ-FiveM-Server#70] [Q&A] [answered]
Bonjour,

C'est en lien avec la discussion : https://github.com/SOZ-Faut-etre-Sub/SOZ-FiveM-Server/issues/69

Comment faire pour effectuer un pull request ? Il me demande de choisir entre deux branches mais dans la liste je n'ai pas le choix.

https://puu.sh/JxxNN/3d2b5cdcb7.png

De débute sur Github et je demande votre aide ^^"

Si c'est trop compliquer à expliquer, je laisserais quelqu'un le faire à  ma place.

Merci :)

**Comment by Vincent-Sniteur** [comment, marked as answer]
Hello, désolé pour la réponse tardive.

Tu trouveras toutes les explications dans ce tuto : [Tuto Fork & Pull Request](https://grafikart.fr/tutoriels/fork-pull-request-591)
